docs/testing/python.md

```diff
@@ -74,6 +85,57 @@ The default_matter_test_main() function is used to run the test on the command
 line. These two lines should appear verbatim at the bottom of every python test
 file.
 
+## Defining the test arguments
+
+Below is the format:
+
+```
+# test-runner-runs: <run_identifier>
+# test-runner-run/<run_identifier>/app: ${TYPE_OF_APP}
```

Comment:
a description somewhere that the placeholders here are taken from the yaml input may help, that way people know how to add new placeholders.

Reply:
Added details of the placeholders in the description section below to the arguments example section.